packages/react-centra-checkout/src/Context/index.tsx

```diff
@@ -232,34 +227,35 @@ export function CentraProvider(props: ProviderProps) {
   }
 
   const selectionApiCall = useCallback(
-    async (
-      apiCall:
-        | Promise<CheckoutApi.Response<CheckoutApi.SelectionResponse>>
-        | (() => Promise<CheckoutApi.Response<CheckoutApi.SelectionResponse>>),
-    ) => {
+    async (apiCall: Promise<unknown> | (() => Promise<unknown>)) => {
       window.CentraCheckout?.suspend()
       const response = typeof apiCall === 'function' ? await apiCall() : await apiCall
 
-      if ('selection' in response && response.selection) {
-        setSelection(response)
+      if (
+        Boolean(response) &&
+        response !== null &&
+        typeof response === 'object' &&
```

Comment:
suggestion: As we now have a `isPlainObject` predicate in `@noaignite/utils` which already is a dependency of this package, we might as well replace these 3 lines with the predicate :)

Reply:
Done, @maeertin!